e2e/create-cluster_test.go

```diff
@@ -428,6 +433,11 @@ var _ = Describe("CreateCluster", func() {
 			"--control-plane-machine-count=1",
 			"--worker-machine-count=1",
 			"--from", "templates/cluster-template.yaml")
+		cmd.Env = append(os.Environ(),
+			"NODE_VM_IMAGE_TEMPLATE=quay.io/capk/ubuntu-container-disk:20.04",
+			"IMAGE_REPO=k8s.gcr.io",
+			"CRI_PATH=/var/run/containerd/containerd.sock",
+		)
```

Comment:
these env changes need to be modified in `hack/functest.sh` rather than being in the e2e test